nuntium/templates/nuntium/profiles/stats.html

```diff
@@ -0,0 +1,39 @@
+{% extends "base_edit.html" %}
+{% load i18n %}
+{% load pagination_tags %}
+{% load nuntium_tags %}
+{% load staticfiles %}
+
+{% block extrajs %}
+{% endblock extrajs %}
+
+{% block extracss %}
+<link href="https://gitcdn.github.io/bootstrap-toggle/2.2.0/css/bootstrap-toggle.min.css" rel="stylesheet">
+{% endblock extracss %}
+
+{% block extrascripts %}
```

Comment:
What's the difference between blocks 'extrajs' and 'extrascripts'? Is it just two different places you can add javascript in? I wonder if there is a better name for them if so.